librz/asm/arch/hexagon/hexagon.h

```diff
@@ -106,8 +107,10 @@ typedef struct {
 	char mnem_infix[128]; // The mnemonic without the pre- and postfix.
```

Comment:
Could you please convert other members' comments of this struct to Doxygen format too?